### ISSUE TITLE
tvOS version

### ISSUE THREAD

**Issue by skyfliers** [WilliamLCobb/iNDS#15]
Probably not high on your to-do list, but eventually it would be nice to have iNDS work on the new apple tv.

**Comment by pmp174** [comment]
If there is an issue with using the touch screen then maybe the siri remote could be used for touch or be used as a cursor to touch sort of like the way the wii menu is controlled  if possible.

**Comment by WilliamLCobb** [comment]
A developer of Nitrogen is working on a tvOS version right now. I would recommend using that and maybe he'll be willing to merge it into iNDS when he finishes

**Comment by pmp174** [comment]
I remember seeing it but I can't find his fork. That would maybe complicate things a bit. Since INDS while having similar code has went through a couple versions and changes which would need to be accounted for when merging. I think a better idea would be to ask him if he can begin transferring his tvos classs to INDS in his own fork and then ask him to merge. I will start looking for his fork so that I can try to contact him.

**Comment by pmp174** [comment]
https://github.com/NitrogenEmulator/Nitrogen/tree/46dcb2f5256c9ca1a94e70dc7432723173ceca72
I found the fork I'll try to contact him

**Comment by pmp174** [comment]
Good news

![image](https://cloud.githubusercontent.com/assets/15956420/11945364/230cc202-a81d-11e5-8b71-7d263717b6d1.png)

**Comment by WilliamLCobb** [comment]
Awesome. I'll add that soon. I wish I had an Apple TV to test it on though

**Comment by pmp174** [comment]
Ill be buying one soon so I can also help test

**Comment by pmp174** [comment]
I got the apple tv , I am going to start trying to port some of the code from the other project for tv os on my own fork. Once ive added it , I will make a pull request. It really can't be too hard they wrote most of it 😂. Today I will hunt for a usb c to usb a cable.

**Comment by pmp174** [comment]
![image](https://cloud.githubusercontent.com/assets/15956420/12076195/c1cdf942-b16e-11e5-94b2-6e048ecc85e3.jpg)
Found it

**Comment by pmp174** [comment]
@WilliamLCobb I have ported most of the code, I will keep updating my fork to add more TV OS implementation . I am not 100% sure the devs actually made the project work well with the siri remote but I will keep on adding things. I think this will work best with a mfi controller. ( icade is not supported on tv os since its a keyboard ). If you want to check what Ive done you could check my fork. https://github.com/pmp174/iNDS

**Comment by DarknesGaming** [comment]
iNDS will be working with Apple TV? That is AWESOME!
Does that involve making a TV version of iNDS though? I'm not sure if an app developed on the iOS SDK in Xcode will work on tvOS.
For now, I really wish AirPlay was possible on the iNDS. I really want to see my games on my iPhone be displayed on the big screen because I'm bored.

**Comment by pmp174** [comment]
@DarknesGaming airplay is possible on inds

**Comment by DarknesGaming** [comment]
It is? How do I enable that?

**Comment by pmp174** [comment]
Just turn airplay on both devices on as usual and try to connect to the device. Like if you have an iphone use controll center and click on the apple tv from the airplay

**Comment by Anuskuss** [comment]
When you do this, it will display the top screen on your AirPlay device (and hides it on your iDevice). There are currently no settings, but it works and this is awesome.

**Comment by pmp174** [comment]
yes and if you fix the layout profile you can make the touch screen the iOS device and the tv makes it awesome. look at how I did it. https://www.youtube.com/watch?v=E8V4igX5Glk

**Comment by DarknesGaming** [comment]
Good to know how to do all of this.
